CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to BGPMon
+## Coding standards
+We use the [Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments) as the basic standard for all of our code.
+In addition to this, we have supplemental standards defined below.
+
+### Gofmt and golint
+All code should be formatted with gofmt and pass golint without any errors. This helps to avoid small formatting errors and lack
+of documentation.
+
+### Import Style
+All imports within the project should be contained within an import() block, even if there's only one. Gofmt will sort imports
+within a group alphabetically. Import groups are separated by a newline. There should only be 3 import groups, appearing in order.
+
+1. Standard library
+2. Local project packages
+3. 3rd party libraries
+
+Some files may not use all of the groups above. Here is an example of the import style:
+
+    import (
+        "fmt"
+        "os"
+
+        "github.com/CSUNetSec/bgpmon/util"
+
+        "github.com/spf13/cobra"
+    )
+    
+### Variables
+Multi-word variables should use camelCase. The first letter should not be capitalized, as that creates an exported name in golang.
+For consistency, even variables within functions, that are not exported, should use the same style. This also applies to fields
+within structs, and unexported function declarations. Exported names should use CamelCase, with the first letter capitalized.
+
+    func unexportedFunc() {}
+    func ExportedFunc() {}
+    
+    struct example {
+        exampleInt int
+    }
+
+### Declarations in if statements
+In golang, it is possible to declare mulitple variable in the conditional line of an if statement. The scope of this variable 
+is the extent of the if and any attached else clauses.
+
+    if a, b := foo; a != b {
+        // a and b are in scope here
+    } else {
+        // Also in scope here
+    }
+    // Neither are in scope here
+This type of statement is **not** allowed in BGPMon. Instead, the following style is preferred:
```

Comment:
I was under the impression that we settled on this standard specifically just for error handling rather than a general thing, as described [here](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow).  At the same time I don't think this situation happens for any non-error types anywhere in the project, so it doesn't really matter that much.

Reply:
We can discuss this more in the review session this week, but I'd prefer to make our standards as general as possible, rather than defining them for each use case. As you said, this doesn't really create an issue for the rest of the project, so I'll leave this as-is for now.